pallets/dispatcher/src/lib.rs

```diff
@@ -91,6 +100,10 @@ pub mod pallet {
 	#[pallet::getter(fn aave_manager_account)]
 	pub type AaveManagerAccount<T: Config> = StorageValue<_, T::AccountId, ValueQuery, T::DefaultAaveManagerAccount>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn account_extra_gas)]
+	pub type AccountExtraGas<T: Config> = StorageMap<_, Blake2_128Concat, EvmAddress, u64, OptionQuery>;
```

Comment:
no need for map, can be just one value shared

Reply:
hm. that;s true

Reply:
and then i dont have "issues" with address mappings, as i dont need any adresss, and Ican get rid of some additional requirements due to the AsRef

much better

Reply:
done